### PR TITLE
Add rhumba feedstock

### PR DESCRIPTION
**Pull request by marimeireles**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by marimeireles** [comment]
@wolfv ?
Can we merge it?

File: recipes/rhumba/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+{% set sha256 = "4126489d3b8c8f1d94c09dac126ee527a352874c72e615d9f02d9fba1e24b29e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/marimeireles/rhumba/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/rhumba/meta.yaml:12]: Write the sha directly here

File: recipes/rhumba/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{% set name = "rhumba" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "4126489d3b8c8f1d94c09dac126ee527a352874c72e615d9f02d9fba1e24b29e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (xhochy) [inline recipes/rhumba/meta.yaml:10]: No need for this line

**Review by xhochy — COMMENTED** [review]
Minor comments,then this is ready.

**Comment by marimeireles** [comment]
Thanks @xhochy ! ;)

**Pull request merged**